cfme/tests/automate/test_common_methods.py

```diff
@@ -180,3 +180,99 @@ def test_miq_password_decrypt(klass):
         execute_methods=True,
     )
     assert result.validate()
+
+
+@pytest.fixture(scope='module')
+def original_class(domain):
```

Comment:
@ganeshhubale I think you can use this global fixture in place of this
https://github.com/ManageIQ/integration_tests/blob/c6913cc5bf0ff2f0c8186c5a84be0f9ea4a8e620/cfme/fixtures/automate.py#L40